app/models/lifetime_usage.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class LifetimeUsage < ApplicationRecord
+  include Currencies
+  include Discard::Model
+  self.discard_column = :deleted_at
+
+  belongs_to :organization
+
+  validates :currency, inclusion: {in: currency_list}
+
+  validates :current_usage_amount_cents, numericality: {greater_than_or_equal_to: 0}
+  validates :invoiced_usage_amount_cents, numericality: {greater_than_or_equal_to: 0}
+
+  monetize :current_usage_amount_cents,
+    :invoiced_usage_amount_cents,
+    with_model_currency: :currency
+
+  default_scope -> { kept }
+
+  def subscription
+    Subscription.active
+      .joins(:organization)
+      .where(organizations: {id: organization_id})
+      .where(external_id: external_subscription_id)
+      .sole
```

Comment:
do we want this method to raise an exception if there is no subscription or more than one matching subscription?

Reply:
Yes, it is expected. Since we will delete the lifetime usage along with the subscription, we are expecting a lifetime usage to always belongs to an existing active subscription